spec/abci/README.md

```diff
@@ -7,14 +7,15 @@ parent:
 
 # ABCI
 
+ABCI stands for "**A**pplication **B**lock**c**hain **I**nterface".
 ABCI is the interface between Tendermint (a state-machine replication engine)
 and your application (the actual state machine). It consists of a set of
-_methods_, where each method has a corresponding `Request` and `Response`
-message type. Tendermint calls the ABCI methods on the ABCI application by sending the `Request*`
-messages and receiving the `Response*` messages in return.
+_methods_, each with a corresponding `Request` and `Response`message type. 
+To perform state-machine replication, Tendermint calls the ABCI methods on the 
+ABCI application by sending the `Request*` messages and receiving the `Response*` messages in return.
 
-All message types are defined in a [protobuf file](https://github.com/tendermint/tendermint/blob/master/proto/tendermint/abci/types.proto).
```

Comment:
Removed in favor of not linking against master of a different repository.

Reply:
This makes sense, but it would be nice if there were some indication to the reader of where to go for the definitions. Do we have a place on the docs site we could publish the release version of the proto file? 

Or if that isn't practical, maybe it's still worth saying something like

> All message types are defined in `abci/types.proto` in the [tendermint repository](https://github.com/tendermint/tendermint).

or words to that effect?

Reply:
Realized that they're also in this repo! linked to them here.